docs/guides/graph-in-google-sheets.md

```diff
@@ -1,3 +1,6 @@
+---
+meta_description: "Image-charts offers several guides such as setup a dependency graph in Google Sheets with image-charts."
```

Comment:
Here is how to display a (dependency) graph inside Google Sheets thanks to Image-Charts API